spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceReader.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.conformance
+
+import com.typesafe.config.Config
+import org.slf4j.Logger
+import za.co.absa.enceladus.conformance.config.ConformanceConfigInstance
+import za.co.absa.enceladus.conformance.interpreter.{FeatureSwitches, ThreeStateSwitch}
+
+class ConformanceReader(log: Logger, conf: Config) {
```

Comment:
The name here seems misleading. You speak about a reader but there is none. 

And there is no reason to call it the same as in Standardization. Which at least has one method to produce a reader.

Reply:
I would say that it is still a reader because it reads parameters

Reply:
I am knowledgable about Enceladus in general and when I saw `ConformanceReader` I see a `DataFrameReader` that reads the parquet produced by Std. And it is hardly a ConfigReader when it is passed to it. 

Maybe `ConformancePropertiersX`? Maybe `ConformancePropertiersHandler`

Reply:
If it is to be changed, then it could be `ConformancePropertiesProvider`, but it seems quite long

Reply:
I think that could be better. Even though it is long.

Reply:
Thinking about it, since it is in package conformance, do you need the `Conformance` prefix?

Reply:
Good point, I can just call it PropertiesProvider